enums.py

```diff
@@ -65,6 +66,8 @@ class Tablenames(Enum):
     LABELING_TASK_LABEL = "labeling_task_label"
     EMBEDDING = "embedding"
     EMBEDDING_TENSOR = "embedding_tensor"
+    EMBEDDER = "embedder"
+    EMBEDDER_PAYLOAD = "embedder_payload"
```

Comment:
new tables without alembic file?

Reply:
again, there was something wrong with the PR. I delete this